README.md

```diff
@@ -0,0 +1,368 @@
+In order to compile Charm++ into dynamic libraries, one needs to specify the
+`-build-shared` option to the Charm compiler script `charmc`
+at link time. For example, to compile Charm++ under `netlrts-linux/tmp`, run
+
+     $ make charm++ OPTS='-O -build-shared'
+
+Charm++'s dynamic libraries are compiled into the `lib_so/` directory.
+Typically, they are generated with a `.so` suffix.
+
+Note that the `-build-shared` option is automatically turned on when building
+Charm++ using the `build` script. So you don't need to pass `-build-shared`
+option to "build".
+
+One can compile a Charm++ application linking against Charm++ dynamic
+libraries by linking with `charmc`'s `-charm-shared` option.
+For example:
+
+     $ charmc -o pgm pgm.o -charm-shared
+
+You can then run the program as usual.
+Note that linking against Charm++ dynamic libraries produces much smaller
+binaries and takes much less linking time.
+
+
+## For More Information
+
+The Charm++ documentation is at https://charm.readthedocs.io/
+
+The Charm++ web page, with more information, more programs,
+and the latest version of Charm++, is at	http://charmplusplus.org
+
+The UIUC Parallel Programming Laboratory web page, with information
+on past and present research, is at http://charm.cs.illinois.edu
+
+The Charm++ mailing list, for questions, comments, suggestions,
+improvements, or bug reports is charm@cs.illinois.edu
+
+
+## Authors
+Charm++ was created and is maintained by the Parallel Programming Lab,
+in the Computer Science department at the University of Illinois at
+Urbana-Champaign.  Our managing professor is Dr. L.V. Kale; students
+and staff have included (in rough time order) Wennie Shu, Kevin Nomura, Wayne
+Fenton, Balkrishna Ramkumar, Vikram Saletore, Amitabh B. Sinha, Manish
+Gupta, Attila Gursoy, Nimish Shah, Sanjeev Krishnan, Jayant DeSouza,
+Parthasarathy Ramachandran, Jeff Wright, Michael Lang, Jackie Wang,
+Fang Hu, Michael Denardo, Joshua Yelon, Narain Jagathesan, Zehra Sura,
+Krishnan Varadarajan, Sameer Paranjpye, Milind Bhandarkar, Robert Brunner,
+Terry Wilmarth, Gengbin Zheng, Orion Lawlor, Celso Mendes, Karthik Mahesh,
+Neelam Saboo, Greg Koenig, Sameer Kumar, Sayantan Chakravorty, Chao Huang,
+Chee Lee, Fillipo Gioachin, Isaac Dooley, Abhinav Bhatele, Aaron Becker,
+Ryan Mokos, Ramprasad Venkataraman, Gagan Gupta, Pritish Jetley, Lukasz
+Wesolowski, Esteban Meneses, Chao Mei, David Kunzman, Osman Sarood,
+Abhishek Gupta, Yanhua Sun, Ehsan Totoni, Akhil Langer, Cyril Bordage,
+Harshit Dokania, Prateek Jindal, Jonathan Lifflander, Xiang Ni,
+Harshitha Menon, Nikhil Jain, Vipul Harsh, Bilge Acun, Phil Miller,
+Seonmyeong Bak, and Karthik Senthil. Current developers include Eric Bohm,
+Ronak Buch, Michael Robson, Eric Mikida, Sam White, Juan Galvez, Nitin Bhat,
+Kavitha Chandrasekar, Jaemin Choi, Matthias Diener, Evan Ramos,
+Raghavendra Kanakagiri, and Venkatasubrahmanian Narayanan.
+
+    Copyright (C) 1989-2019 Regents of the University of Illinois
```

Comment:
this renders as if the text was `Copyright ...`, not sure if this is intentional

Reply:
I intentionally did this to set it off, but perhaps it's best left as just regular text; I'll make that change.